src/vs/editor/contrib/find/findWidget.ts

```diff
@@ -807,6 +809,9 @@ export class FindWidget extends Widget implements IOverlayWidget, IHorizontalSas
 
 		this._matchesCount = document.createElement('div');
 		this._matchesCount.className = 'matchesCount';
+
+		this._matchesCount.setAttribute('aria-live', 'assertive');
```

Comment:
If we use `aria.alert`, then this is no longer necessary, right?

Reply:
yes